src/bin/list_store.rs

```diff
@@ -0,0 +1,299 @@
+// "List Store" gtk3 demo ported to gtk-rs by Ömer Sinan Ağacan <omeragacan@gmail.com>
+
+extern crate gio;
+extern crate gtk;
+
+use gio::*;
+use gtk::*;
```

Comment:
Don't do wildcard imports for the whole crates, see my comment in the other example